integration/kubernetes/init.sh

```diff
@@ -17,6 +17,10 @@ source "${SCRIPT_PATH}/../../.ci/lib.sh"
 # FIXME: Issue: https://github.com/clearcontainers/tests/issues/934
 sudo iptables -P FORWARD ACCEPT
 
+# Remove existing CNI configurations:
+sudo rm -r /var/lib/cni/networks/*
+sudo rm -r /etc/cni/net.d/*
```

Comment:
This might fail if those folders are empty, I think adding `-f` should help here:
```bash
sudo rm -rf /var/lib/cni/networks/*
sudo rm -rf /etc/cni/net.d/*
```